docs/Using Fleet/Audit-logs.md

```diff
@@ -627,9 +627,9 @@ This activity contains the following fields:
 }
 ```
 
-### Type `created_macos_profile`
```

Comment:
Are these types returned in the `GET /activities` API as `type` properties? If so, these would be breaking changes. 

Again, we could do it but we have to be really intentional about it. 

Check to make sure we're confident that customers and users aren't building automations w/ this. And if they are, is the change worth making them update?

I think it's unlikely they, because MDM is relatively new and our MDM customers don't seem to be integrating with `GET /activities`

Reply:
I know those types are automatically generated but they are all specific to `macos` profiles. I guess we could have new cross platform types in parallel with old `macos` specific types, which we can deprecate at some point?

Reply:
> I guess we could have new cross platform types in parallel with old macos specific types, which we can deprecate at some point?

This is what I was thinking too. It's similar to what we're doing with the other macOS endpoints. Let's bring this up with Rachael Shaw.

Reply:
After discussion with George, we decided to add new types for Windows profiles and display in activity feed that Windows profile has been added, deleted or edited.

Reply:
@noahtalerman  Would it be better to outline these in the requirements section of the issue, so we don't need to remove them?